src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java

```diff
@@ -1,18 +1,41 @@
 package uk.gov.companieshouse.insolvency.delta.mapper;
 
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import org.mapstruct.AfterMapping;
 import org.mapstruct.Context;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
 import uk.gov.companieshouse.api.delta.Insolvency;
 import uk.gov.companieshouse.api.insolvency.InternalCompanyInsolvency;
+import uk.gov.companieshouse.api.insolvency.InternalData;
+
 
 @Mapper(componentModel = "spring", uses = {CaseMapper.class})
 public interface InsolvencyMapper {
 
-    @Mapping(target = "etag", ignore = true) // doesn't exist on source
-    @Mapping(target = "status", ignore = true) // doesn't exist on source
-    @Mapping(target = "deltaAt", source = "deltaAt")
-    @Mapping(target = "cases", source = "caseNumbers")
+    @Mapping(target = "externalData.etag", ignore = true) // doesn't exist on source
+    @Mapping(target = "externalData.status", ignore = true) // doesn't exist on source
+    @Mapping(target = "internalData.deltaAt", ignore = true)
+    @Mapping(target = "externalData.cases", source = "caseNumbers")
     InternalCompanyInsolvency insolvencyDeltaToApi(Insolvency insolvency,
                                                    @Context String companyNumber);
+
+    /**
```

Comment:
I dowloaded the code and when I tried to mvn compile, it gave me a check style violation. 

Please see it below:
[WARNING] src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java:[26] (javadoc) SummaryJavadoc: First sentence of Javadoc is incomplete (period is missing) or not present.

Can you please confirm if you have the same problem?

Reply:
https://ci.platform.aws.chdev.org/teams/team-tiny-rebels/pipelines/insolvency-delta-consumer/jobs/analyse-pull-request/builds/29

Yep - fails in CI pipeline

Reply:
Should be resolved now